Headers/DebugServer2/GDBRemote/Session.h

```diff
@@ -143,8 +143,8 @@ class Session : public SessionBase {
   void Handle_qOffsets(ProtocolInterpreter::Handler const &,
                        std::string const &);
   void Handle_qP(ProtocolInterpreter::Handler const &, std::string const &);
-  void Handle_qPlatform_IO_MkDir(ProtocolInterpreter::Handler const &,
-                                 std::string const &);
+  void Handle_qPlatform_mkdir(ProtocolInterpreter::Handler const &,
```

Comment:
Can you make sure this is not a case of gdb vs. lldb? I find it surprising that we were handling a packet named differently, with a different format, etc.